src/main/java/seedu/duke/command/FindCommand.java

```diff
@@ -29,11 +27,20 @@ public FindCommand(String keywords) {
 
     @Override
     public String execute() {
+        String notes = "";
+
         ArrayList<Note> filteredNotes = (ArrayList<Note>) notebook.getNotes().stream()
                 .filter((s) -> s.getTitle().contains(keywords))
                 .collect(Collectors.toList());
 
-        printFindNotesMessage(filteredNotes);
-        return null;
+        for (int i = 0; i < filteredNotes.size(); i++) {
+            notes += (i + 1) + "." + filteredNotes.get(i).toString();
+        }
+
+        if (filteredNotes.isEmpty()) {
+            return "There are no matching notes. Please try another search query.\n";
+        }
+
+        return "Here are the matching notes in your list:\n" + notes;
```

Comment:
Can use InterfaceManager.LS (line separator) for the \n to standardize all new line